Composer/packages/client/src/components/CreationFlow/CreateOptions.tsx

```diff
@@ -265,6 +274,14 @@ export function CreateOptions(props: CreateOptionsProps) {
       <DialogWrapper isOpen {...dialogWrapperProps} dialogType={DialogTypes.CreateFlow} onDismiss={onDismiss}>
         <ChoiceGroup label={choiceGroupTitle} options={choiceOptions} selectedKey={option} onChange={handleChange} />
         <h3 css={listHeader}>{formatMessage('Examples')}</h3>
+        {featureFlags?.REMOTE_TEMPLATE_CREATION_EXPERIENCE?.enabled && (
```

Comment:
Add a TODO comment here about removing this code once the feature flag is removed.